cw/__init__.py

```diff
@@ -20,35 +22,62 @@
           'rabbit-exchange': os.getenv('LCW_RABBIT_EXCHANGE', 'local.lcmap.changes.worker'),
           'rabbit-result-routing-key': os.getenv('LCW_RABBIT_RESULT_ROUTING_KEY', 'change-detection-result'),
           'rabbit-ssl': os.getenv('LCW_RABBIT_SSL', False),
-          'ubid_band_dict' : {
+          'api-host': os.getenv('LCW_API_HOST', 'http://localhost:5678'),
+          'ubid_band_dict': {
               'tm': {'red': 'band3',
                      'blue': 'band1',
                      'green': 'band2',
                      'nirs': 'band4',
-                     'swirs1': 'band5',
-                     'swirs2': 'band7',
+                     'swir1s': 'band5',
+                     'swir2s': 'band7',
                      'thermals': 'band6',
                      'qas': 'cfmask'},
               'oli': {'red': 'band4',
                       'blue': 'band2',
                       'green': 'band3',
                       'nirs': 'band5',
-                      'swirs1': 'band6',
-                      'swirs2': 'band7',
+                      'swir1s': 'band6',
+                      'swir2s': 'band7',
                       'thermals': 'band10',
                       'qas': 'cfmask'}}}
 
+numpy_type_map = {
+    'UINT8': np.uint8,
+    'UINT16': np.uint16,
+    'INT8': np.int8,
+    'INT16': np.int16
+}
+
+
+def spectral_map(cfg):
+    """ Return a dict of sensor bands keyed to their respective spectrum """
+    _spec_map = dict()
+    _map = {'thermal': 'toa -11', 'cfmask': '+cfmask -conf'}
+    for bnd in 'blue green red nir swir1 swir2'.split(' '):
+        _map[bnd] = 'sr'
+
+    for spectra in _map:
+        url = "{host}/landsat/tile-specs?q=((tags:{band}) AND tags:{spec})"
+        resp = requests.get(url.format(host=cfg['api-host'],
+                                       spec=spectra,
+                                       band=_map[spectra])).json()
+        _spec_map[spectra] = [i['ubid'] for i in resp]
+    return _spec_map
+
 
 def send(cfg, message):
     return messaging.send(cfg, message)
 
+
 def listen(cfg, callback):
     messaging.listen(cfg, callback)
 
+
 def launch_task(cfg, msg_body):
```

Comment:
i left this here because it opens us up to being able to run pyccd on spark via a shell script if we want to (i think for debug, etc, we'll want to at some point.)